ten-simple-rules-dockerfiles.tex

```diff
@@ -290,38 +290,37 @@ \section*{Introduction}\label{introduction}}
 sharing of the computing environment, on the other hand, is enabled by
 containerization, which allows for documenting and sharing entire
 workflows in a comprehensive way. This sharing of computational assets
-is paramount to increase reproducibility of computational research.
-While ``papers'' based on the traditional journal article format can
-share high level details of research, computational research is
-oftentimes way too complicated to be disseminated alongside in this
+is paramount to increasing the reproducibility of computational
+research. While ``papers'' based on the traditional journal article
+format can share high level details of research, computational research
+is oftentimes way too complicated to be disseminated alongside in this
 format {[}1{]}. In that the actual contribution to knowledge includes
 the full computing environment that produced a result {[}2{]},
 containerization is needed.
 
 Containerization plays an important role in providing instructions for
 packaging the building blocks of computer-based research (i.e.~code,
-data, documentation, and the computing environment). While a container
-cannot package all of these blocks, it represents a human- \emph{and}
-machine-readable recipe to capture the computing environment and
+data, documentation, and the computing environment). Containers are
+built from plain text files which represent a human- \emph{and}
```

Comment:
_files that represent_

Reply:
This is the same point which has now been addressed in the `rmd` file above.